### PR TITLE
Ignores through front matter

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
No longer need to maintain a list of build ignores.

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -33,43 +33,29 @@
...
+  ignores = ["/blog/de/*", "/blog/nl/*", "/jobs/*"]
+end
+
+ignores.each do |path|
+  ignore path
+end
+
+ready do
+  ignores = sitemap.resources.select do |i|
+    i.path =~ /\.html/ &&
```
Comment (fatboypunk) [inline config.rb]: isn't i just the resource? why don't we name it so?

Comment (fatboypunk) [reply config.rb]: Do we need to check if the path contains .html? or is just checking if there is a `lang_ignore` good enough?
What other pages do we state lang_ignore where we don't want to ignore it

File: config.rb [hunk 2]
```diff
@@ -33,43 +33,29 @@
...
+      i.data&.lang_ignore.to_s.include?(I18n.locale.to_s)
+  end
+
+  ignores.each do |resource|
+    ignore resource.path
```
Comment (fatboypunk) [inline config.rb]: Why do we loop through the resources again to ignore it? can't we do that in the loop above here?

File: config.rb (hunk 1 above)
Comment (sn3p) [reply config.rb]: If you skip the html check you'll also check agains images, scripts, stylesheets, fonts, etc.

A waste of resources I guess :D

Comment (fatboypunk) [reply config.rb]: You're doing that here anyway right?

File: config.rb (hunk 2 above)
Comment (RonaldDijkstra) [reply config.rb]: No reason to double it no.

File: config.rb (hunk 1 above)
Comment (fatboypunk) [reply config.rb]: May leave this as is

File: config.rb [hunk 3]
```diff
@@ -33,28 +33,24 @@
...
+    if resource.data&.lang_ignore.to_s.include?(I18n.locale.to_s)
+      ignores << resource.path
+    end
   end
 
-  ignores.each do |resource|
-    ignore resource.path
+  ignores.each do |path|
+    ignore path
   end
```
Comment (fatboypunk) [inline config.rb]: I don't think this is needed
you could just do this where you're now doing `ignores << resource.path`

File: config.rb [hunk 4]
```diff
@@ -48,8 +48,7 @@
...
 
   sitemap.resources.each do |resource|
-    if resource.path =~ /\.html/ &&
-       resource.data&.lang_ignore.to_s.include?(I18n.locale.to_s)
```
Comment (fatboypunk) [inline config.rb]: no need to remove this

Comment (RonaldDijkstra) [reply config.rb]: Haha ok just a sec :)

Comment (RonaldDijkstra) [reply config.rb]: Removed the remove :)

**Pull request merged**